content/3_Silver/Prefix_Sums.mdx

```diff
@@ -700,13 +700,13 @@ A common application of prefix sums is to determine, given a set of intervals, t
 
 In the focus problem, we are given $n$ intervals $[a,b]$, and we want to find the maximum number of intervals that cover any particular point.
 
-A naïve approach to this problem is to create an array $\texttt{ctr}$ where $\texttt{ctr}$ is the number of intervals that cover each point $i$. We can do this by looping through each interval $[a,b]$ and increasing $\texttt{ctr}[i]$ by $1$ for each index in $a \leq i \leq b$.
+A naïve approach to this problem is to create an array $\texttt{ctr}$, where $\texttt{ctr}[i]$ is the number of intervals that cover each point $i$. We can do this by looping through each interval $[a,b]$ and increasing $\texttt{ctr}[i]$ by $1$ for each index in $a \leq i \leq b$.
 
 This results in a $\mathcal{O}(nV)$ time complexity (where $0 \leq a \leq b \leq V$), which easily times out (think what happens when the interval $[0, V]$ is queried $n$ times).
 
 We can do better. It's easy to see that an increment of $x$ (before computation) in $\texttt{arr}[i]$ causes all subsequent $\texttt{prefix}[i...V]$ (after computation) to increase by $x$. We can also "undo" this operation by adding $-x$ to $\texttt{arr}[i]$. This concept can be conceptualized through increment and decrement points. An increment point increases (and decrement decreases) all subsequent cells. Note that our decrement point is located at $B+1$ because the interval is inclusive - decrementing at point $B$ turns the interval to $[A, B)$.
 
-Example 1: Add two to each point in the interval $[2, 4]$
+**Example 1**: Add two to each point in the interval $[2, 4]$
 
 Our matrix after adding 2 at our increment (start) point (before computation)
```

Comment:
huh

Reply:
What's wrong here?

Reply:
why is it matrix

Reply:
Oh, I see. For some reason i thought the comment was referring to line 709